etc/profile-m-z/mutt.profile

```diff
@@ -10,13 +11,14 @@ noblacklist /var/mail
 noblacklist /var/spool/mail
 noblacklist ${HOME}/.Mail
 noblacklist ${HOME}/.bogofilter
-noblacklist ${HOME}/.cache/mutt
+noblacklist ${HOME}/.config/mutt
```

Comment:
There's a `blacklist ${HOME}/.cache/mutt` in `disable-programs.inc`. We should keep the `noblacklist` for cases like the users adds `ignore private-cache`. Or is `${HOME}/.cache/mutt` not used?

Reply:
On first run it didn't create a cache directory. I assume it is for mail cache which the users can specify the directory. For that use case I can add it back.